.php_cs

```diff
@@ -1,91 +0,0 @@
-<?php
-$finder = Symfony\CS\Finder\DefaultFinder::create()
-    ->files()
-    ->name('*.php')
-    ->in(__DIR__ . '/src')
-    ->in(__DIR__ . '/tests')
```

Comment:
Could possibly remove the test files from the run though to reduce the amount of time it needs to process.